features/bootstrap/FeatureContext.php

```diff
@@ -85,6 +85,12 @@ public function createDatabase()
         ]);
         $output = new BufferedOutput();
         $application->run($input, $output);
+
+        $input = new ArrayInput([
+            'command' => 'demo:create_data',
```

Comment:
I also recommend to avoid to load global data for your tests and instead load specific data you need for your scenarios. It's way easier to manage. If you change the data loaded globally, it can very easily break a lot of tests that were depending on it, if your data is specific for each scenario, it can only break that specific scenario

Reply:
That's true.

However, we have too few developers to afford such additional work right now.